### PR TITLE
[#5294] Add topic property command to Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add topic property command to Gravitino CLI

### Why are the changes needed?

To expand Gravitino CLI.

Fix: #5294

### Does this PR introduce _any_ user-facing change?

No but it adds additional command to the Gravitino CLi.

### How was this patch tested?

Tested locally with a running Apache Kafka instance.

### CODE REVIEW

**Comment by justinmclean** [comment]
@shaofengshi once the first topic PR and the owner PR is merged this is ready to review or you could just review the topic property parts

**Comment by justinmclean** [comment]
@shaofengshi this is ready to review

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/RemoveTopicProperty.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      String property) {
+    super(url, ignoreVersions);
+    this.metalake = metalake;
+    this.catalog = catalog;
+    this.schema = schema;
+    this.topic = topic;
+    this.property = property;
+  }
+
+  /** Remove a property of a schema. */
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/RemoveTopicProperty.java:67]: typo: "schema" --> "topic"

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+  protected final String metalake;
+  protected final String catalog;
+  protected final String schema;
+  protected final String topic;
+  protected final String property;
+  protected final String value;
+
+  /**
+   * Set a property of a schema.
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java:42]: typo: "schema" --> "topic"

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    super(url, ignoreVersions);
+    this.metalake = metalake;
+    this.catalog = catalog;
+    this.schema = schema;
+    this.topic = topic;
+    this.property = property;
+    this.value = value;
+  }
+
+  /** Set a property of a schema. */
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java:71]: same here

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTopicProperties.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      String schema,
+      String topic) {
+    super(url, ignoreVersions);
+    this.metalake = metalake;
+    this.catalog = catalog;
+    this.schema = schema;
+    this.topic = topic;
+  }
+
+  /** List the properties of a catalog. */
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListTopicProperties.java:64]: typo: "catalog" --> "topic"

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java (hunk 2 above)
Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetTopicProperty.java:42]: Done

**Comment by justinmclean** [comment]
@shaofengshi all fixed

**Review by shaofengshi — APPROVED** [review]
LGTM

**Pull request merged**